### PR TITLE
Fixed useless compilation SASS when entry points were not found.

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #48.

### CODE REVIEW

File: lib/compiler/utils.js [hunk 1]
```diff
@@ -288,6 +288,16 @@ require( [ 'tests' ], bender.defer(), function( err ) {
 		}
 
 		function renderThemeFromEntryPoints( callback ) {
+			if ( paths.length === 0 ) {
+				gutil.log( `Entry points for '${ gutil.colors.cyan( fileName ) }' have not found... Skipped.` );
+
+				stream.push( new gutil.File( {
```
Comment (Reinmar) [inline lib/compiler/utils.js]: What do you actually put to the stream if there are no entry points? This code deserves some comment.

File: lib/compiler/utils.js [hunk 2]
```diff
@@ -288,6 +288,16 @@ require( [ 'tests' ], bender.defer(), function( err ) {
 		}
 
 		function renderThemeFromEntryPoints( callback ) {
+			if ( paths.length === 0 ) {
+				gutil.log( `Entry points for '${ gutil.colors.cyan( fileName ) }' have not found... Skipped.` );
```
Comment (Reinmar) [inline lib/compiler/utils.js]: "have not been found"

File: tests/compiler.js [hunk 3]
```diff
@@ -180,17 +180,23 @@ describe( 'compiler', () => {
 	} );
 
 	describe( 'process.sass()', () => {
+		let stream;
+		let files = [];
+
+		beforeEach( () => {
+			stream = require( 'stream' );
```
Comment (Reinmar) [inline tests/compiler.js]: Why requireing in `beforeEach`?

Comment (pomek) [inline tests/compiler.js]: Why not? `stream` is required for the whole group `process.sass()`.

File: lib/compiler/utils.js (hunk 1 above)
Comment (pomek) [inline lib/compiler/utils.js]: Fixed. This file isn't needed.

File: tests/compiler.js (hunk 3 above)
Comment (Reinmar) [inline tests/compiler.js]: Because all other `require()` calls are at the beginning. And it makes no sense to introduce beforeEach block for a next one.

Comment (pomek) [inline tests/compiler.js]: Moved at the beginning.

**Pull request merged**